spec/acceptance/apt_keyring_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+PUPPETLABS_FILE_CHECK_COMMAND = 'ls /etc/apt/keyrings | grep \'puppetlabs-keyring.gpg\''
```

Comment:
Should use https://serverspec.org/resource_types.html#file for this. Or maybe better would be to call something like `gpg --show-keys` on the file and verify that it outputs info about the expect key.

Reply:
> Should use https://serverspec.org/resource_types.html#file for this. Or maybe better would be to call something like `gpg --show-keys` on the file and verify that it outputs info about the expect key.

Hi @kenyon, Thanks for the PR review. While there are 3 points I would need your clarification/details/insights :
1.  I dont think **gpg --show-keys** would work because gpg package doesnt get installed by itself with apt. I tried that by provisioning the docker machine and running the acceptance test and got the message **E: E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation .**
2. Anyways, the keyring file would contain gpg value in binary format. Not sure how to compare it with the expected key (even what would be expected key? )
3. As an alternate, We could try deleting the keyring file through shell , which will ensure that file was present at the location ..Thoughts ??

Happy to connect over community slack, if need further discussions